~

```diff
@@ -0,0 +1,30 @@
+pick 8e73ecf23f [feature][trino-python-client] add new trino python client && support trino set session
```

Comment:
I think this file came by a mistake. Please check.

Reply:
> I think this file came by a mistake. Please check.

Sry  I will remove this file

Reply:
This file has been removed, may caused by git rebase